src/test/ui/pure-sum.rs

```diff
@@ -19,7 +17,7 @@ fn sums_to(v: Vec<isize> , sum: isize) -> bool {
 
 fn sums_to_using_uniq(v: Vec<isize> , sum: isize) -> bool {
     let mut i = 0;
-    let mut sum0: Box<_> = box 0;
+    let mut sum0: Box<_> = 0.into();
```

Comment:
Why are sum of the tests switching to `.into()` and not `Box::new`?

Reply:
In this instance it's clear due to the `: Box<_>`. At the start I have sometimes added into because it's shorter, but later on I went on to using Box::new for most things.

Reply:
@Mark-Simulacrum should I change the into invocations to Box::new?

Reply:
I would've preferred that but I don't think it's really that necessary, and I don't want you to redo all that work another time.